Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm

```diff
@@ -131,6 +137,13 @@
     m_mockAccessibilityElement = WTFMove(mockAccessibilityElement);
 
     accessibilityTransferRemoteToken(accessibilityRemoteTokenData());
+
+    // Close Mach connection to Launch Services.
+    // FIXME: Replace the constant 4 with kLSServerConnectionStatusReleaseNotificationsMask when available in the SDK.
```

Comment:
I think it would be tidier to have a define for `kLSServerConnectionStatusReleaseNotificationsMask` protected by something like `#if !HAVE(LS_DONOTCONNECTMASK)`. That way the code using the definition never has to change, just a future removal of the local define.

Reply:
Fixed!

Thanks for reviewing!